tests/remote_point_evaluation/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+CMAKE_MINIMUM_REQUIRED(VERSION 3.1.0)
```

Comment:
Why do you want another directory for the tests here that doesn't match the location of the corresponding source files?

Reply:
I would like to keep it this way to be able to collect the test that use `RemotePointEvaluation` in one place.